client/src/app/containers/Game/slice.js

```diff
@@ -50,7 +50,9 @@ const gameSlice = createSlice({
     refreshGame: {
       reducer: (state, action) => {
         if (state.currentGameID !== action.payload.id) {
-          throw `bad game id: expected "${state.currentGameID}", got "${action.payload.id}"`;
+          throw Error(
+            `bad game id: expected "${state.currentGameID}", got "${action.payload.id}"`,
+          );
```

Comment:
Have you done any more research about `throw`ing in a reducer?

Reply:
:nope: :noidea: